awx/api/templates/api/group_all_hosts_list.md

```diff
@@ -1,4 +1,4 @@
-# List All {{ model_verbose_name_plural|title }} for this {{ parent_model_verbose_name|title }}:
```

Comment:
`anora` is a new filter you'll see sprinkled everywhere that adds "a or an" before the variable; it looked nicer in the generated docs than "for this X"